scripts/snyk-test.js

```diff
@@ -6,12 +6,25 @@ const { glob } = require('glob');
 const { promisify } = require('util');
 const execFile = promisify(childProcess.execFile);
 
+async function fileExists(filePath) {
+  try {
+    await fs.access(filePath);
+    return true;
+  } catch {
+    return false;
+  }
+}
+
 async function snykTest(cwd) {
   const tmpPath = path.join(os.tmpdir(), 'tempfile-' + Date.now());
 
   let execErr;
 
   try {
+    if (!(await fileExists(path.join(cwd, `package.json`)))) {
+      return;
```

Comment:
You're pushing `null` onto results again.

Reply:
nope, that's ok, the `"null"` issue, only happens with the monorepo root, but in the monorepo root a package.json is granted to be there, cause that package.json is being read to fetch workspaces. The issue is not in this function in any case.

Reply:
https://github.com/mongodb-js/compass/blob/594d5049270abd08869276da331732010b792afd/scripts/snyk-test.js#L81